src/components/Lessons/MonthDays.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+
+export default function MonthDays(props) {
+
+    const [days, setDays] = useState([])
+    const [curDate, setCurDate] = useState('')
+    const [isMounted, setIsMounted] = useState(false)
+    
+    useEffect(() => {
+        console.log(props.curDate)
+        setCurDate(props.curDate)
+        console.log(Date.parse(props.curDate))
+        // getCurWeekDates()
+        if (!isMounted) {
+
+
+            setIsMounted(true)
+        }
+    })
+
+    const getCurWeekDates = (date) => {
```

Comment:
Another example of a helper method that should not be part of the component. These are details that make it harder for me to read the important component code and understand what it does